src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -213,11 +204,11 @@ bool MulticopterLandDetector::_get_maybe_landed_state()
 	}
 
 	// Next look if all rotation angles are not moving.
-	float maxRotationScaled = _params.maxRotation_rad_s * landThresholdFactor;
+	float maxRotationScaled = math::radians(_param_lndmc_rot_max.get()) * landThresholdFactor;
```

Comment:
@dagar, the issue you highlighted is addressed here.

Reply:
Flight test land detection works properly in the and the value checks out when queried: https://review.px4.io/plot_app?log=9c628e6e-b90e-4d1b-9308-e6b5a8cc15b5